core/kernel/dt_driver.c

```diff
@@ -253,8 +280,8 @@ void *dt_driver_device_from_node_idx_prop(const char *prop_name,
 	int idx = 0;
 	int idx32 = 0;
 	int prv_cells = 0;
-	int phandle_node = 0;
-	uint32_t phandle = -1;
+	uint32_t phandle = 0;
+	int phandle_node = -1;
```

Comment:
fixup to squash into commit "core: dt_driver: add phandle node and fdt to dt_driver_phandle_args".

Reply:
Arg, my bad.